make/tests.mk

```diff
@@ -113,56 +125,56 @@ rr_graph.xml: rr_graph.real.xml
 
 ##########################################################################
 ##########################################################################
-
+ifeq($(SOURCES),)
 SOURCE_E = $(wildcard *.eblif)
 SOURCE_V = $(filter-out %_tb.v,$(wildcard *.v))
-SOURCE = $(basename $(SOURCE_V)$(SOURCE_E))
-$(info SOURCE = $(SOURCE))
-
-SOURCE_F = $(abspath $(SOURCE_V)$(SOURCE_E))
-
 SOURCES = $(SOURCE_E) $(SOURCE_V)
+else
+SOURCE_E = $(filter %.eblif,$(SOURCES))
+SOURCE_V = $(filter %.v,$(SOURCES))
+endif
 
-ifneq ($(words $(SOURCES)),1)
 ifeq ($(words $(SOURCES)),0)
 $(error "No sources found!")
 endif
-$(error "Multiple sources found! $(SOURCES)")
```

Comment:
Should we add a check for the case of multiple eblif files?

Reply:
Done.